### PR TITLE
Removed fulltags in nanoserver dockerfile, replaced with 1809 value

### PR DESCRIPTION
**Pull request by anamnavi**
## PR Summary

changing windows images to use generated full tag, and not set the full tag dynamically. Thus, a dockerfile is created for each nanoserver's (or whichever distribution's) version. This is PR is just to test it for version 1809.

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
will update tag
  - [x] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/docs/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

**Comment by anamnavi** [comment]
Nice to meet you @RDIL :) I'm a new developer on the PowerShell team. I submitted the PR to get feedback from @TravisEz13 as I resolve the related issue #490. It's a work in progress at the moment but thanks for assigning the tag to it!

**Comment by TravisEz13** [comment]
I don't see any issues.  Now just to duplicate the folder for the other tags

**Comment by RDIL** [comment]
Nice to meet you too @anamnavi :)

File: release/preview/nanoserver1903/meta.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        }
+    ],
+    "tagTemplates": [
+        "#psversion#-nanoserver-#tag#",
+        "preview-nanoserver-#shorttag#"
+    ],
+    "Base64EncodePackageUrl": true,
+    "TestProperties": {
+        "size": 1
+    }
```
Comment (TravisEz13) [inline release/preview/nanoserver1903/meta.json:17]:
```suggestion
    },
    "UseAcr": true
```

**Comment by TravisEz13** [comment]
Last commit failed because our normal CI system cannot run anything but 1809.
I added an attribute in the Meta.json to move it to the other CI system.

**Comment by anamnavi** [comment]
Ah I see @TravisEz13 . What does ACR stand for? Just for my understanding

**Comment by RDIL** [comment]
Azure Container Registry @anamnavi

**Review by RDIL — APPROVED** [review]
LGTM

**Comment by anamnavi** [comment]
Each nanoserver version folder has a dependabot folder with a docker file in it. For [example](https://github.com/anamnavi/PowerShell-Docker/blob/windows-fulltag-issue/release/preview/nanoserver1809/dependabot/Dockerfile). In previous implementation it was specifying dummy docker image to get as the parent image as the latest nanoserver version (2004). Should I change it to be the specific version associated with the folder instead? (i.e dependabot dockerfile in the 1809 folder gets dummy docker image of nanoserver1809)

**Comment by TravisEz13** [comment]
We should remove the depandabot dummy files and adjust the dependabot config to point to the 2004 version.  There is no point to point depandabot at the rest as they are already out of date.

**Comment by anamnavi** [comment]
At the moment it's pointed to 1809 [here ](https://github.com/PowerShell/PowerShell-Docker/blob/master/.github/dependabot.yml#L37). I'll modify it to point to 2004. I'm assuming this is the dependabot config you'd meant.

**Comment by anamnavi** [comment]
I removed the dependabot docker files from nanoserver1809, nanoserver1903, nanoserver1909 folders. I left the dependabot folders for consistency with how the other distributions/versions' folders are structured, but I can remove this. I left nanoserver2004/dependabot/dockerfile as is because it gets referenced in the dependabot.yml file I updated to point there.

File: release/preview/nanoserver1903/getLatestTag.ps1 [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation.
```
Comment (TravisEz13) [inline release/preview/nanoserver1903/getLatestTag.ps1:1]: delete this file in the nanoserver folders

Comment (TravisEz13) [inline release/preview/nanoserver1903/getLatestTag.ps1:1]: Delete this file in all the preview nanoserver folders

**Review by TravisEz13 — CHANGES_REQUESTED** [review]
see comment

**Comment by RDIL** [comment]
@PoshChan please retry CI

**Comment by PoshChan** [comment]
@RDIL, successfully started retry of `docker-PR`

**Comment by RDIL** [comment]
The CI is failing due to timeouts, not sure what's going on there.

**Comment by anamnavi** [comment]
Yeah, I'm not sure about the timeouts either. It's failing because of timeouts for the release>stable builds, and the release>lts builds, which didn't have files modified for. It also cancelled the release>preview build (which I did modify files for) so I just re-ran those tests specifically.

**Review by RDIL — APPROVED** [review]
Other than the failing build, all looks good.

File: release/preview/nanoserver1809/meta.json [hunk 3]
```diff
@@ -5,23 +5,15 @@
...
-        "#psversion#-nanoserver-#tag#",
+        "#psversion#-nanoserver1809-#tag#",
         "preview-nanoserver-#shorttag#"
     ],
     "Base64EncodePackageUrl": true,
     "TestProperties": {
         "size": 1
-    }
+    },
+    "UseAcr": true
```
Comment (TravisEz13) [inline release/preview/nanoserver1809/meta.json:18]:
```suggestion
    }
```

1809 should not use ACR as it can run in normal CI

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

File: release/preview/nanoserver1903/getLatestTag.ps1 (hunk 2 above)
Comment (anamnavi) [reply release/preview/nanoserver1903/getLatestTag.ps1]: deleted

File: release/preview/nanoserver1809/meta.json (hunk 3 above)
Comment (anamnavi) [reply release/preview/nanoserver1809/meta.json]: removed

File: release/preview/nanoserver1809/meta.json [hunk 4]
```diff
@@ -1,23 +1,14 @@
...
-        {
-            "Tag": "1909"
-        },
-        {
-            "Tag": "2004"
         }
     ],
     "tagTemplates": [
-        "#psversion#-nanoserver-#tag#",
+        "#psversion#-nanoserver1809-#tag#",
```
Comment (TravisEz13) [inline release/preview/nanoserver1809/meta.json:11]:
```suggestion
        "#psversion#-nanoserver-#shorttag#",
```

File: release/preview/nanoserver1903/meta.json [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    "IsLinux": false,
+    "PackageFormat": "PowerShell-${PS_VERSION}-win-x64.zip",
+    "osVersion": "Nano Server, version 1903",
+    "shortTags": [
+        {
+            "Tag": "1903"
+        }
+    ],
+    "tagTemplates": [
+        "#psversion#-nanoserver1903-#tag#",
```
Comment (TravisEz13) [inline release/preview/nanoserver1903/meta.json:11]:
```suggestion
        "#psversion#-nanoserver-#shorttag#",
```

File: release/preview/nanoserver1909/meta.json [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    "IsLinux": false,
+    "PackageFormat": "PowerShell-${PS_VERSION}-win-x64.zip",
+    "osVersion": "Nano Server, version 1909",
+    "shortTags": [
+        {
+            "Tag": "1909"
+        }
+    ],
+    "tagTemplates": [
+        "#psversion#-nanoserver1909-#tag#",
```
Comment (TravisEz13) [inline release/preview/nanoserver1909/meta.json:11]:
```suggestion
        "#psversion#-nanoserver-#shorttag#",
```

File: release/preview/nanoserver2004/meta.json [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    "IsLinux": false,
+    "PackageFormat": "PowerShell-${PS_VERSION}-win-x64.zip",
+    "osVersion": "Nano Server, version 2004",
+    "shortTags": [
+        {
+            "Tag": "2004"
+        }
+    ],
+    "tagTemplates": [
+        "#psversion#-nanoserver2004-#tag#",
```
Comment (TravisEz13) [inline release/preview/nanoserver2004/meta.json:11]:
```suggestion
        "#psversion#-nanoserver-#shorttag#",
```

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**